ckanext/restricted/tests/test_plugin.py

```diff
@@ -436,3 +438,64 @@ def test_restricted_string_snippet(self):
             '{}">{}</a>'.format(org['name'], org['title'])
         )
         assert expected in html4
+
+    @mock.patch('ckan.lib.mailer.mail_recipient')
+    def test_all_org_admins_are_emailed_on_request_access(self, mocked_mail_recipient, app):
+
+        # create two admins and one regular member of an org
+        user_1 = factories.User(email='user_1@example.com')
+        user_2 = factories.User(email='user_2@example.com')
+        user_3 = factories.User(email='user_3@example.com')
```

Comment:
if you name the variables differently the comment "# create two admins and one regular member of an org" won't be needed
```
admin1 = factories
admin2 = factories
member = factories
```